### DISCUSSION TITLE
Question about filtering reads prior to Mixcr

### DISCUSSION THREAD

**Discussion by SirKuikka** [milaboratory/mixcr#1675] [Q&A] [answered]
Hi,

If I have already aligned my reads against the human reference genome using Dragen, is it a good idea to first take reads with samtools that align to chr7 and chr14 , and then provide this as input to Mixcr. I am only interested in TCRs with alfa and beta chains. The run time was significantly faster when done this way. And one test I did showed that the number of clones I got out from Mixcr was quite similar with and without filtering.

**Comment by mizraelson** [comment, marked as answer]
Yes, prefiltering the target chromosomes where the TCR locus is located will speed up the process. There shouldn't be any losses as you only filter out non target reads.
